bin/bootstrax

```diff
@@ -445,6 +445,17 @@ def main_loop():
                 new_runs_seen += 1
                 process_run(rd)
                 continue
+        else:
+            # We are on an old eb with not so much to do, perhaps one of
+            # the veto systems needs processing?
+            rd = consider_run(
+                {'detectors': {'$in': ["muon_veto", "neutron_veto"]},
```

Comment:
This will match linked-mode runs, which maybe we don't want. To avoid this the query should look more like `{'detectors': {'$ne': 'tpc'}}`, but `$ne` isn't a particularly efficient query.

Reply:
@darrylmasson , thanks, good catch. 

On the bootstrax side, inefficiency is not a problem as this is on the old ebs anyway. Problem is of course that this operator is executed on the mongo side of the story.

Reply:
From the docs:
> The inequality operator $ne is not very selective since it often matches a large portion of the index. As a result, in many cases, a $ne query with an index may perform no better than a $ne query that must scan all documents in a collection